test/test-marshal.js

```diff
@@ -3,7 +3,7 @@
 import { test } from 'tape-promise/tape';
 import harden from '@agoric/harden';
 import { makeMarshal, mustPassByPresence } from '@agoric/marshal';
-import { maybeExtendPromise } from '@agoric/eventual-send';
+import '@agoric/eventual-send';
```

Comment:
Let's mark this as `// for side-effect: ensure HandledPromise is installed`, since we're no longer invoking anything we import. Also, please double-check that `node -r esm ./node_modules/.bin/tape test/test-marshal.js` still works (I think we added the `maybeExtendPromise()` so that each test file could be run independently, without relying upon the order of other test files being executed.

Reply:
Added and verified.